docs/src/man/getting_started.md

```diff
@@ -558,6 +558,21 @@ Equivalently, the `in` function can be called with a single argument to create
 a function object that tests whether each value belongs to the subset
 (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+!!! note
+
+    While data frames support indexing syntax that is very similar to
+    matrices, subsetting from a `DataFrame` will return a copy, not
+    a view.
+
+    The only subsetting situations where data frames will **not** return a copy are:
+
+    - when a `!` is placed in the first indexing position (`df[!,:A]`, or `df[!, [:A, :B]]`),
+    - when using `.` notation (`df.A`),
```

Comment:
maybe better write `getproperty`?

Reply:
I don't think most people know what `getproperty` refers to (I don't!). I can put `getproperty` in parentheses?

Reply:
That is why I have written to wait for someone else to comment. I can make sure that the text is correct with respect to the implementation and it would be good to have someone else to comment on the wording :smile:.